libs/platform/src/lib/components/form/form.spec.ts

```diff
@@ -88,7 +88,7 @@ describe('Simple Form', () => {
             imports: [
                 ReactiveFormsModule,
                 FdpFormGroupModule,
-                FdpInputModule,
+                PlatformInputModule,
```

Comment:
if PlatformInputModule module is not added the test case fails, this test cases were already existing, since renaming was done from FdpInputModule to PlatformInputModue. This change was introduced.

Reply:
Not sure how this was moved down here. This comment belongs to the input module